Simplemania

RipballOsu opened the pull request:
<img src="https://media.discordapp.net/attachments/608070341461540864/813623841485750312/unknown.png">

Same pull request, now with rebase from master.

A simple mania overlay, I'm bad with names...
This overlay only works for mania mode, V2 Score it won't show up, in goosu future updates will add that.
Progress bar is not working properly, since sometimes it gets Infinity value.

I think all the current img mods for mania are there (1K-10K,Co-op,Random,FadeIn, etc...)

RipballOsu committed:
Simplemania

Rebase

Dartandr commented:
1) Some characters bigger than np line like @ in that case 
![image](https://user-images.githubusercontent.com/32641958/109597113-a7429900-7b28-11eb-8166-6e21a6a15465.png)

Dartandr commented:
2) About imgs 
I don't understand why you decided to use imgs on mods where you have only text.
Also why don't you use html and css instead of mania.png. Image in that case cause visual troubles like bad smoothing on text 
![image](https://user-images.githubusercontent.com/32641958/109597932-4bc4db00-7b29-11eb-81f5-926fcf1fe3ae.png)

Dartandr commented:
3) I recommend to add countUp to all of those 
![image](https://user-images.githubusercontent.com/32641958/109598159-c8f05000-7b29-11eb-9f58-fed9dd36b995.png)

Dartandr commented:
4) Also i think it would be great to add some fade in/out animation on counter appearance

Dartandr commented:
Not sure about the design itself but i guess it's some kind of replica of another mania like game. So would leave it for l3lackShark

Dartandr commented:
In index.js everything looks good to me

RipballOsu committed:
Updated some changes

Remove all images and display it as css/html.

- Add some values to countUp.
- Add overlay animation after/before gameplay.

RipballOsu commented:
I think now is fixed, it took me some time to make mania.png as only html/css, can you check if everything is okay?
Thanks.

RipballOsu committed:
Update index.js

Fix index.js (it doesnt show up the overlay properly)

RipballOsu committed:
Fixes

- Remove Song Info (by BlackShark suggestion)
- Fixed in order to work with in game overlay.

RipballOsu commented:
New fixes.
- Sometimes the overlay didnt show up even the player was on mania mode.
- Change the main overlay resolution.
![image](https://media.discordapp.net/attachments/731854916934565953/817670123455381534/screenshot1826.jpg)

Dartandr commented on SimpleMania/index.js:
```diff
@@ -0,0 +1,280 @@
+let tempOD;
+let tempHP;
+let tempCS;
+let tempPos;
```
clean up your code from unused variables

Dartandr commented on SimpleMania/css/style.css:
```diff
@@ -0,0 +1,223 @@
+    padding-right: 5px !important;
+    margin-left: -5px !important;
+    margin-right: -5px !important;
+}
```
unused

Dartandr commented on SimpleMania/css/style.css:
```diff
@@ -0,0 +1,223 @@
+    font-size: 20px;
+    font-family: "Sica 3", Helvetica, sans-serif;
+    font-weight: normal;
+    color: white;
+    width: 460px;
+    height: 550px;
+    padding: 20px;
+    position: relative;
```
highly recommend to remove those and right,bottom in #box_right cause it's useless in that case

Dartandr commented on SimpleMania/css/style.css:
```diff
@@ -0,0 +1,223 @@
+#box_right {
+	width: 400px;
+	height: 550px;
+    right: 40px;
+    bottom: 20px;
```
.

Dartandr commented on SimpleMania/css/style.css:
```diff
@@ -0,0 +1,223 @@
+    text-shadow: -1px 1px 2px #FFF, 1px 1px 2px #FFF, 1px -1px 0 #FFF, -1px -1px 0 #FFF;
+    color:rgb(65,105,225) ;
+}
+#h100,#h50,#h0,[id='300'] ,#geki,#katu,#pp,#ur
```
Would be better to add some class for this

Dartandr commented on SimpleMania/css/style.css:
```diff
@@ -0,0 +1,223 @@
+    padding: 30px; 
+}
+
+#Mar,#Per,#Gre,#Gud,#Ba,#Mi,#RR,#Scc,#MD,#ppcont,#urcont
```
Same, need some class

Dartandr commented on SimpleMania/index.html:
```diff
@@ -0,0 +1,57 @@
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="css/style.css">	
+    <link rel="stylesheet" href="css/odometr.css">
+    <script src="deps/odometr.js"></script>
```
unused

Dartandr commented on SimpleMania/index.html:
```diff
@@ -0,0 +1,57 @@
+    <script src="deps/odometr.js"></script>
+    <script src="deps/countUp.js"></script>
+    <script src="deps/reconnecting-websocket.min.js"></script>
+    <script src="deps/Chart.min.js"></script>
+    <script src="deps/smooth.js"></script>
```
unused

Dartandr reviewed (changes requested):
You need to remove all unused variables, scripts, etc/
Not the best way how you decided to make a html and css alignment but fine for now. Would be better if you made a one div for each line and position elements inside each line. Than just hardcode position of all elements.

RipballOsu committed:
Fixes

- Removed all js unused code
- HTML for alignment.
- CSS fixed.

RipballOsu commented:
I think now is ok.
- Removed all unused code (js/html)
- And check all the css/html code.

Thanks

RipballOsu reviewed (commented):
All checked.

RipballOsu reviewed (commented):
Oops, I didnt check all the files in the git changes.

Dartandr commented:
add you counter width height and screenshot to readme.md

Dartandr commented:
And also you need to change font cause it's non-free. Preferably to use monospaced fonts
![image](https://user-images.githubusercontent.com/32641958/110542521-327bea00-813a-11eb-9639-7c152eefd426.png)

RipballOsu committed:
Font

Updating font and readme

RipballOsu committed:
Update README.md

Repair conflicts with readme.

RipballOsu committed:
Update README.md

RipballOsu committed:
Merge branch 'master' into master

RipballOsu commented:
Updated font (I checked the font was free) and add the counter in readme.md

Dartandr committed:
Merge branch 'master' into master
